### PR TITLE
Changelog update - 7.3.1

### PR DESCRIPTION
**Pull request by dprokop**
```

  /*** Grafana changelog  ***/


  ### Bug Fixes
  * **Loki**: Fix error when some queries return zero results. [#28645](https://github.com/grafana/grafana/pull/28645), [@ivanahuckova](https://github.com/ivanahuckova)
  * **PanelMenu**: Fixes panel submenu not being accessible for panels close to the right edge of the screen. [#28666](https://github.com/grafana/grafana/pull/28666), [@torkelo](https://github.com/torkelo)
  * **Plugins**: Fix descendent frontend plugin signature validation. [#28638](https://github.com/grafana/grafana/pull/28638), [@wbrowne](https://github.com/wbrowne)
  * **Prometheus**: Fix copy paste behaving as cut and paste. [#28622](https://github.com/grafana/grafana/pull/28622), [@aocenas](https://github.com/aocenas)
  * **StatPanel**: Fixed value being under graph and reduced likley hood for white and dark value text mixing. [#28641](https://github.com/grafana/grafana/pull/28641), [@torkelo](https://github.com/torkelo)
  * **TextPanel**: Fixes problems where text panel would show old content. [#28643](https://github.com/grafana/grafana/pull/28643), [@torkelo](https://github.com/torkelo)

✔ Generating changelog
```

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,13 @@
+# 7.3.1 (2020-10-30)
+
+  ### Bug Fixes
+  * **Loki**: Fix error when some queries return zero results. [#28645](https://github.com/grafana/grafana/pull/28645), [@ivanahuckova](https://github.com/ivanahuckova)
+  * **PanelMenu**: Fix panel submenu not being accessible for panels close to the right edge of the screen. [#28666](https://github.com/grafana/grafana/pull/28666), [@torkelo](https://github.com/torkelo)
+  * **Plugins**: Fix descendent frontend plugin signature validation. [#28638](https://github.com/grafana/grafana/pull/28638), [@wbrowne](https://github.com/wbrowne)
+  * **Prometheus**: Fix copy paste behaving as cut and paste. [#28622](https://github.com/grafana/grafana/pull/28622), [@aocenas](https://github.com/aocenas)
+  * **StatPanel**: Fix value being under graph and reduced likley hood for white and dark value text mixing. [#28641](https://github.com/grafana/grafana/pull/28641), [@torkelo](https://github.com/torkelo)
```
Comment (aknuds1) [inline CHANGELOG.md:8]:
```suggestion
  * **StatPanel**: Fix value being under graph and reduced likelihood for white and dark value text mixing. [#28641](https://github.com/grafana/grafana/pull/28641), [@torkelo](https://github.com/torkelo)
```

**Review by aknuds1 — CHANGES_REQUESTED** [review]
Where's the PR fixing the CloudWatch time series duplication bug?

**Comment by dprokop** [comment]
@aknuds1 - it was missing add to changelog label. Fixing

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,13 @@
+# 7.3.1 (2020-10-30)
+
+  ### Bug Fixes
+  * **Loki**: Fix error when some queries return zero results. [#28645](https://github.com/grafana/grafana/pull/28645), [@ivanahuckova](https://github.com/ivanahuckova)
```
Comment (dprokop) [inline CHANGELOG.md:4]:
```suggestion
  * **Cloudwatch**: Fix duplicate metric data. [#28642](https://github.com/grafana/grafana/pull/28642), [@zoltanbedi](https://github.com/zoltanbedi)
  * **Loki**: Fix error when some queries return zero results. [#28645](https://github.com/grafana/grafana/pull/28645), [@ivanahuckova](https://github.com/ivanahuckova)
```

**Review by aknuds1 — APPROVED** [review]
LGTM

**Pull request merged**